shoop/campaigns/models/basket_conditions.py

```diff
@@ -0,0 +1,88 @@
+# This file is part of Shoop.
+#
+# Copyright (c) 2012-2016, Shoop Ltd. All rights reserved.
+#
+# This source code is licensed under the AGPLv3 license found in the
+# LICENSE file in the root directory of this source tree.
+from django.db import models
+from django.utils.encoding import force_text
+from django.utils.translation import ugettext_lazy as _
+from polymorphic.models import PolymorphicModel
+
+from shoop.core.fields import MoneyValueField
+from shoop.core.models import Product
+from shoop.utils.properties import MoneyPropped, PriceProperty
+
+
+class BasketCondition(PolymorphicModel):
+    model = None
+    active = models.BooleanField(default=True)
+    name = _("Basket condition")
+
+    def matches(self, basket, lines):
+        return False
+
+    @property
+    def description(self):
+        return _("Define the required %s for this campaign." % self.name.lower())
```

Comment:
Translators does not have any idea what these %s contains. Use %(name)s instead?